### PR TITLE
Add std to cargo template in generate-metadata

### LINKED ISSUE #1254: undefined reference to `__ink_generate_metadata' again
[This](https://github.com/paritytech/cargo-contract/issues/909) bug has reappeared due to a change to the template cargo.toml

Offending commit:
https://github.com/paritytech/cargo-contract/commit/94d96c5df60dc43d01f7329dbcef99c9454e9314

`std` needs to be enabled here:

https://github.com/paritytech/cargo-contract/blob/94d96c5df60dc43d01f7329dbcef99c9454e9314/crates/build/templates/generate-metadata/_Cargo.toml#L13

### PR DESCRIPTION
**Pull request by forgetso**
Fixes #1254

### CODE REVIEW

File: crates/build/templates/generate-metadata/_Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ name = "metadata-gen"
 path = "main.rs"
 
 [dependencies]
-contract = { path = "../.." }
+contract = { path = "../..", features= ["std"] }
```
Comment (ascjones) [inline crates/build/templates/generate-metadata/_Cargo.toml:13]: `std` should be already enabled via the `default` feature below?

**Review by ascjones — COMMENTED** [review]
Indeed this is how it originally was but I moved this to the `default` feature (see below), so it should be enabled (we are not invoking `cargo run` with `--no-default-features`).

I am unable to reproduce the error, can you provide a reproduction so I can understand what is going on?

**Comment by ascjones** [comment]
@dependabot appears to be drunk and merged this with 0 changes, not sure what happened there.

**Comment by goastler** [comment]
I don't recall force-pushing to this, I have no idea what's happened here! I've been making some changes on our fork of cargo contract today, but these were very minor (added a file and ~5 lines changed in another) and had no involvement with the master branch on our end nor parity's.

Genuinely no idea what's happened here!

**Comment by ggwpez** [comment]
The MR has +-0 and 0 commits since you force-pushed a master commit. It therefore breaks and is marked as "merged" although nothing actually was merged.

**Pull request merged**